### PR TITLE
Adding moment for Moyal distribution and corresponding tests

### PR DESCRIPTION
**Pull request by jlopezarriaza**
Add Moyla moment and corresponding tests to help with https://github.com/pymc-devs/pymc/issues/5078

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -3727,6 +3727,14 @@ def dist(cls, mu=0, sigma=1.0, *args, **kwargs):
 
         return super().dist([mu, sigma], *args, **kwargs)
 
+    def get_moment(rv, size, mu, sigma):
+        mu, sigma = at.broadcast_arrays(mu, sigma)
```
Comment (ricardoV94) [inline pymc/distributions/continuous.py]:
```suggestion
```
This is not needed here. The parameters will be automatically broadcasted below as long as they show up in the same expression

**Review by ricardoV94 — COMMENTED** [review]
Looks great. Thanks for opening the PR.I have just a small change suggestion.

File: pymc/distributions/continuous.py (hunk 1 above)
Comment (jlopezarriaza) [reply pymc/distributions/continuous.py]: Removed. Thanks.

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Looks good. Waiting to see if the tests pass

**Comment by ricardoV94** [comment]
Thanks for the help @jlopezarriaza!

**Pull request merged**